lib/graphql/plug/endpoint.ex

```diff
@@ -9,12 +9,13 @@ defmodule GraphQL.Plug.Endpoint do
       {mod, func} -> apply(mod, func, [])
       s -> s
     end
-    %{schema: schema}
+    root_callback = Keyword.get(opts, :root, %{})
```

Comment:
Don't think `root_callback` is an appropriate name in this case, since you could either pass a root value directly (ie no function call) or a function which returns the value. `root_value` is used in https://github.com/graphql/express-graphql#options